packages.ini

```diff
@@ -470,6 +475,8 @@ validate_extras = pytest
 [idna==3.3]
 [idna==3.4]
 
+[ijson==3.2.3]
+
```

Comment:
actually ijson is dead and archived too so we definitely should not be using it

for a few 100 MB I doubt it will make a performance difference to just use stdlib json

Reply:
oh maybe not archived -- just the pypi url goes to the old maintainer

Reply:
to be clear here -- ijson is _probably_ fine -- I don't have major concerns with it but it is unlikely to be solving the problem at hand.  `smart-open` isn't getting you anything though -- normal `open` is already streamed unless you call `.read()` on it